css/quiz.css

```diff
@@ -0,0 +1,27 @@
+/*****
+CSS Rules 
+*********/
+
+body {
+	font-family: arial;
+
+}
+
+
+
+nav 
+	{
```

Comment:
Be consistent in your formatting. It's more typical to have your opening curly brace on the same line as the selector, but one line down is acceptable. Just pick on format and use it consistently